project/app/utils/complexity/squad_score.py

```diff
@@ -108,7 +108,7 @@ async def squad_score(document: str, scaler):
         "avg_word_len": 1,
         "quotes_number": 1,
         "unique_words": 1,
-        "adj_num": 1
+        "adj_num": 1,
```

Comment:
Adding in the [PEP8 docs](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas) on the trailing comma for posterity, as this was totally new to me!